### PR TITLE
fix: design fixes for collection layout

### PR DESCRIPTION
**Pull request by dcshzj**
Fixes ISOM-997.

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

File: src/templates/next/layouts/Collection/CollectionClient.tsx [hunk 1]
```diff
@@ -170,6 +196,41 @@ const getPaginatedItems = (
   )
 }
 
+const updateAppliedFilters = (
+  appliedFilters: AppliedFilter[],
+  setAppliedFilters: React.Dispatch<React.SetStateAction<AppliedFilter[]>>,
+  filterId: string,
```
Comment (kishore03109) [inline src/templates/next/layouts/Collection/CollectionClient.tsx:202]: what about 
`setAppliedFilters: (AppliedFilter: AppliedFilter[]) => void,` for less verbose code?

File: src/templates/next/layouts/Collection/CollectionClient.tsx [hunk 2]
```diff
@@ -210,33 +271,71 @@ const CollectionClient = ({
...
+                <div className="flex flex-row flex-wrap gap-3">
+                  {getAppliedFiltersWithLabels(filters, appliedFilters).map(
+                    (appliedFilter) => (
+                      <Pill
+                        key={`${appliedFilter.appliedFilterTypeId}-${appliedFilter.appliedFilterId}`}
+                        content={appliedFilter.label}
+                        onClose={() =>
+                          updateAppliedFilters(
+                            appliedFilters,
+                            setAppliedFilters,
```
Comment (kishore03109) [inline src/templates/next/layouts/Collection/CollectionClient.tsx:327]: hmm why the api so powerful ah? by right it should just remove itself right?

File: src/templates/next/components/internal/Filter/Filter.tsx [hunk 3]
```diff
@@ -14,36 +15,6 @@ const Filter = ({
     filters.reduce((acc, { id }) => ({ ...acc, [id]: true }), {}),
   )
 
-  const updateAppliedFilters = (filterId: string, itemId: string) => {
-    const filterIndex = appliedFilters.findIndex(
```
[the remark this answers is not in the document]
Comment (dcshzj) [reply src/templates/next/components/internal/Filter/Filter.tsx:18]: Hmm this was just moved into CollectionClient.tsx, not removed.

File: src/templates/next/layouts/Collection/CollectionClient.tsx (hunk 1 above)
Comment (dcshzj) [reply src/templates/next/layouts/Collection/CollectionClient.tsx:202]: Updated in b9132a97c2c23386bb81a2a7736c9e2f658d9565!

File: src/templates/next/layouts/Collection/CollectionClient.tsx (hunk 2 above)
Comment (dcshzj) [reply src/templates/next/layouts/Collection/CollectionClient.tsx:327]: Yup it removes itself, this is just passing the arrow function into the Pill component which will only remove itself.

**Review by kishore03109 — APPROVED** [review]

**Pull request merged**